src/target/move.c

```diff
@@ -310,7 +324,18 @@ void move_front(void)
 {
 	enable_walls_control();
 	target_straight(current_cell_start_micrometers, CELL_DIMENSION,
-			get_max_linear_speed());
+			get_max_end_linear_speed());
+	entered_next_cell();
+}
+
+/**
+ * @brief Move front many cells.
+ */
```

Comment:
Include input parameter cells

Reply:
Fixed in 90f10e1963310b0d98664f00a7d0291355aea7ce